portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/UpgradeReport.testcase

```diff
@@ -398,9 +406,17 @@ definition {
 				fail("upgrade.report.warnings is not present in portal log.");
 			}
 
-			if (!(contains(${fileContent}, "upgrade.report.status="))) {
+			if (!(contains(${logContent}, "upgrade.report.status="))) {
```

Comment:
This is being fixed already by https://issues.liferay.com/browse/LPS-182839, added to test with correct format